src/media/css/footer.styl

```diff
@@ -42,54 +36,53 @@ $support-nav-height = 55px;
         p {
             float: left;
             font-size: 16px;
+            padding: 0 0 0 1em;
             width: 60%;
         }
-        ul:not(.support-navigation) {
-            float: right;
-            font-size: 14px;
-            line-height: 1.3;
-            width: 20%;
-
-            a {
-                display: block;
-            }
-        }
         li + li {
             margin-top: 2px;
         }
-        .pad,
-        .support-navigation {
-            margin: 0 auto;
+        li a {
+            display: block;
         }
-        .support-navigation {
-            align-items: baseline;
-            border-bottom: 1px solid $footer-border;
-            display-flex();
-            height: $support-nav-height;
-            margin-top: -20px;
-            max-width: 100%;
-            padding: 0;
-            text-align: center;
+    }
+    .footer-content {
+        margin: 0 auto;
+        padding: 22px 0 0;
+    }
+    .footer-extras {
+        float: right;
+        font-size: 14px;
+        line-height: 1.3;
+        margin: 0;
+        padding-left: 1em;
+        width: 20%;
+    }
+    .footer-support-navigation {
+        align-items: baseline;
```

Comment:
Yeah, I would've used a mixin here to make sure old browsers are addressed @spasovski